action.py

```diff
@@ -211,19 +213,21 @@ def check_commit(urepo, ubranch, target, sha, merge):
     m = re.match(r'^(Revert\s+\")?\[nrf (mergeup|fromtree|fromlist|noup)\]\s+',
                  title)
     if not m:
-        die(f'{sha}: Title does not contain a sauce tag')
+        die(f'{sha}: Title does not contain a sauce tag '
+            f'(mergeup|fromtree|fromlist|noup)')
     revert = m.group(1)
     tag = m.group(2)
     if not tag:
-        die(f'{sha}: Title does not contain a sauce tag')
+        die(f'{sha}: Title does not contain a sauce tag '
+            f'(mergeup|fromtree|fromlist|noup)')
```

Comment:
https://nordicsemi.atlassian.net/wiki/spaces/NCS/pages/108201225/Pull+Requests#PullRequests-Commit(orsauce)tags

Reply:
I'd add:

```suggestion
            f'([mergeup] |[fromtree] |[fromlist] |[noup]) ')
```

Reply:
If [ ] is used, then nrf should be added. And it should be the same in L217 and L222.